### ISSUE TITLE
Does not work on windows

### ISSUE THREAD

**Issue by Cologler** [bensadeh/tailspin#66]
The application panicked (crashed).
Message:  HOME directory not set: NotPresent
Location: 

https://github.com/bensadeh/tailspin/blob/c148d6a72617203318cf126c5e4317697606f0a2/src/theme_io/mod.rs#L14

The `HOME` environment variable name is `USERPROFILE` on Windows.

**Comment by Innocence** [comment]
Just set an environment variable  name **_XDG_CONFIG_HOME**, then it will work.

`-c, --config-path CONFIG_PATH

    Specify the path to a custom configuration file. Defaults to XDG_CONFIG_HOME/tailspin/config.toml or ~/.config/tailspin/config.toml if not set.
`

**Comment by bensadeh** [comment]
Thanks for opening up this issue and letting me know about this. 

In time, I'd like to move away from the UNIX-specific implementation to support to a more open one that supports Windows. However, I don't use Windows myself, so it is not something I'm able to test myself.

**Comment by Cologler** [comment]
@bensadeh Replace to `env::var("HOME").or(env::var("USERPROFILE")).expect("HOME directory not set");` should work.

**Comment by alicekaerast** [comment]
I'd be happy to help test any PRs for this on Windows (though I am away next week)

I tried something similar to @Cologler's suggestion, and it works but now I have further issues:

The way colors are encoded appear to need changing to work with Windows, at least on the AWS SSM session I have:

```
PS D:\software> .\tspin.exe
Missing filename (?[35mtspin --help?[39m for help)
```

It also fails to find `less`:

```
PS D:\software> .\tspin.exe -t D:\apps\logs\web.log.20231109
Failed to open file with less
```

`tail -f D:\apps\logs\web.log.20231109 | .\tspin.exe ` does work though

**Comment by Cologler** [comment]
@alicekaerast Colors work on my environment (windows 11 + Windows Terminal + PowerShell core).

**Comment by alicekaerast** [comment]
Interesting, how are you building it? I'm not sure if that makes any difference, but I have to cross-compile from my Macbook and I used `cargo build --target=x86_64-pc-windows-gnu`. The colors are broken for me on Powershell terminal and command prompt via RDP as well as via SSM connections

I do already have less.exe in my path, so running `less` on its own already works

This is all on Windows Server 2019

**Comment by Cologler** [comment]
> how are you building it?

![image](https://github.com/bensadeh/tailspin/assets/10906962/91ff7930-21e0-49c5-8b80-e86b61d321d5)

**Comment by alicekaerast** [comment]
I built it on my Macbook with the following:

```bash
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh 
rustup target add x86_64-pc-windows-gnu
brew install mingw-w64
cargo build --target=x86_64-pc-windows-gnu
```

I don't have the ability to build it in Windows directly due to network restrictions

**Comment by Cologler** [comment]
@bensadeh Work as expected. Keep this open until release.

**Comment by bensadeh** [comment]
Just released `2.1.0`. Hope everything works well.
